packages/editor/src/envelope/LoadingScreen/LoadingScreen.tsx

```diff
@@ -24,7 +24,7 @@ import { Title } from "@patternfly/react-core/dist/js/components/Title";
 import { Bullseye } from "@patternfly/react-core/dist/js/layouts/Bullseye";
 import { useEditorEnvelopeI18nContext } from "../i18n";
 
-export function LoadingScreen(props: { loading: boolean }) {
+export function LoadingScreen(props: { loading: boolean; style?: string }) {
```

Comment:
```suggestion
export function LoadingScreen(props: { loading: boolean; styleTag?: string }) {
```
Just to make it clear that the parameter corresponds to a tag.

Reply:
@handreyrc I think the commit for this change got overridden on the last force push.

Reply:
@caponetto,

Oh I see, actualy I commited the next change below, however, it uses "style" instead of "styleTag"  and I ended up fixing the name of the prop insted of it's usages.  
It's fixed now!

Please, could you check it again?

Thanks